.github/workflows/calibration.yaml

```diff
@@ -163,6 +163,13 @@ jobs:
           mkdir -p "${task_output_dir}"
 
           python3 "${{ needs.setup.outputs.worktree_path }}/src/scripts/calibration_analyses/analysis_scripts/process.py" "${task_output_dir}" "${RUN_NAME}" "${{ needs.setup.outputs.worktree_path }}/resources"
+
+          # Print info.txt file with some useful information
+          info_txt="${task_output_dir}/info.txt"
+          git -C working-directory log --pretty=format:"%H%x09%ad%x09%s" --date=iso-strict -n 1 > "${info_txt}"
+          echo "${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}" >> "${info_txt}"
```

Comment:
Note: logs are automatically deleted after ~3 months or so, so this line can be useful for reference for some time, but not forever.

Reply:
I think that's okay - it's for an easy way to get to see stdout when something goes wrong.